feature/src/main/java/com/teamdontbe/feature/posting/PostingFragment.kt

```diff
@@ -151,64 +149,37 @@ class PostingFragment : BindingFragment<FragmentPostingBinding>(R.layout.fragmen
 
                 when {
                     etPostingContent.text.toString().length in 1..499 -> {
```

Comment:
```suggestion
                    etPostingContent.text.toString().length in 1..499 -> {
```
P3 : 텍스트 length 상수화하면 유지 보수에 용이할 듯 합니다!

Reply:
상수화 적용했어욤